src/TypinExamples/TypinExamples.TypinWeb/WebConsole.cs

```diff
@@ -9,23 +9,28 @@ public sealed class WebConsole : IConsole
     {
         private readonly IWebTerminal _webTerminal;
         private readonly CancellationToken _cancellationToken;
-        private ConsoleColor foregroundColor;
-        private ConsoleColor backgroundColor;
+
+        private ConsoleColor foregroundColor = ConsoleColor.White;
+        private ConsoleColor backgroundColor = ConsoleColor.Black;
 
         public StreamReader Input { get; }
-        public bool IsInputRedirected { get; }
+        public bool IsInputRedirected => false;
+
         public StreamWriter Output { get; }
-        public bool IsOutputRedirected { get; }
+        public bool IsOutputRedirected => false;
+
         public StreamWriter Error { get; }
-        public bool IsErrorRedirected { get; }
+        public bool IsErrorRedirected => false;
+
         public ConsoleColor ForegroundColor
         {
             get => foregroundColor;
 
             //https://misc.flogisoft.com/bash/tip_colors_and_formatting
             set
             {
-                foregroundColor = value;
+                foregroundColor = value < ConsoleColor.Black || value > ConsoleColor.White ? ConsoleColor.Black : value;
```

Comment:
Should be ConsoleColor.White insted of ConsoleColor.Black, because default foreground color is white

Reply:
Fixed.